extractor/src/main/java/org/schabi/newpipe/extractor/channel/ChannelInfo.java

```diff
@@ -94,6 +81,17 @@ public static ChannelInfo getInfo(ChannelExtractor extractor) throws IOException
             info.addError(e);
         }
 
+        List<ChannelTabInfo> tabs = new ArrayList<>();
+        for (int i = 0; i < extractor.getTabs().size(); i++) {
+            try {
+                ChannelTabInfo tabInfo = ChannelTabInfo.getInfo(extractor.getTabs().get(i), i == 0);
```

Comment:
Is it hardcoded that the first tab will always be the feed one? Also, what happens if the first tab throws an error?

Reply:
Yes, that's hardcodes, see [this conversation](https://github.com/TeamNewPipe/NewPipeExtractor/pull/279#discussion_r394446468), and I'll change that now.